sdk/identity/azure-identity/src/main/java/com/azure/identity/package-info.java

```diff
@@ -2,7 +2,277 @@
- * for retrieving access tokens from various configurations.
+ * <p>The Azure Identity library provides Azure Active Directory (Azure AD) token authentication support across the
+ * Azure SDK. The library focuses on OAuth authentication with Azure AD, and it offers various credential classes
+ * capable of acquiring an Azure AD token to authenticate service requests. All the credential classes in this package
+ * are implementations of the `TokenCredential` interface offered by azure-core, and any of them can be used to
+ * construct service clients capable of authenticating with a `TokenCredential`.</p>
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p>The {@link com.azure.identity.DefaultAzureCredential} is appropriate for most scenarios where the application is
+ * intended to ultimately be run in Azure. This is because the {@link com.azure.identity.DefaultAzureCredential}
+ * combines credentials commonly used to authenticate when deployed, with credentials used to authenticate in a
+ * development environment. Note, this credential is intended to simplify getting started with the SDK by handling
+ * common scenarios with reasonable default behaviors. Developers who want more control or whose scenario isn't
+ * served by the default settings should use other credential types. For more information refer to the
+ * <a href="https://aka.ms/azsdk/java/identity/defaultazurecredential/docs">conceptual knowledge and configuration
+ * details</a>.</p>
+ *
+ * <p><strong>Sample: Construct a simple DefaultAzureCredential</strong></p>
+ *
+ * <!-- src_embed com.azure.identity.credential.defaultazurecredential.construct -->
+ * <pre>
+ * TokenCredential defaultAzureCredential = new DefaultAzureCredentialBuilder&#40;&#41;
+ *     .build&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.identity.credential.defaultazurecredential.construct -->
+ *
+ * <p>The Azure SDK client builders consume TokenCredential for Azure Active Directory (AAD) based authentication.
+ * The TokenCredential instantiated above can be passed into most of the Azure SDK client builders for
+ * AAD authentication.</p>
+ *
+ * <p>The {@link com.azure.identity.DefaultAzureCredential} works well in most of the scenarios as it executes a chain
+ * of credentials underneath which covers well known authentication scenarios for both Azure hosted platforms and
+ * development environment. But, in some scenarios where only a specific authentication mechanism will work, it is
+ * recommended to use that specific credential to authenticate. Let's take a look at the individual
+ * authentication scenarios and their respective credential use below.</p>
+ *
+ * <hr/>
+ *
+ * <h2> Authenticating on Azure Hosted Platforms via Managed Identity</h2>
+ *
+ * <p>The {@link com.azure.identity.ManagedIdentityCredential} authenticates the configured managed identity
+ * (system or user assigned) of an Azure resource. So, if the application is running inside an Azure resource that
```

Comment:
This first paragraph is still a lot to wrap your head around if you are new to identity. It would be great to start with a brief sentence or two on what MI is.